src/Components/VendorCard/VendorCard.scss

```diff
@@ -0,0 +1,25 @@
+.vendor-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  margin-bottom: 35px;
+  height: 420px;
+  width: 400px;
+  border: 3px solid black;
+  border-radius: 10px;
+  background-color: azure;
```

Comment:
Nice work on these cards. I was thinking we could try and implement some mixins to help make our styling consistent and be able to reuse across the app